pyscal/gasoil.py

```diff
@@ -338,8 +338,6 @@ def add_fromtable(
                 self.sgro = sgro_estimate
         if pccolname in dframe:
             # Incoming dataframe must cover the range:
-            if dframe[sgcolname].min() > self.table["SG"].min():
```

Comment:
(this is never reached because there is another test further up that catches it)